terraform/cache/index.html

```diff
@@ -1,59 +1,73 @@
-                </a>
-                </p>
+  <head>
+    <title>cache.nixos.org is up</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://nixos.org/bootstrap/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://nixos.org/bootstrap/css/bootstrap-responsive.min.css"
+    />
+    <style>
+      body {
+        padding-top: 0;
+        margin-top: 4em;
+        margin-bottom: 4em;
+      }
+      body > div {
+        max-width: 800px;
+      }
+      p {
+        text-align: center;
+      }
+      .cache {
+        font-style: italic;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container jumbotron">
+      <div class="jumbotron">
+        <p class="lead">
+          <a href="https://nixos.org/nixos">
+            <img
+              src="https://nixos.org/logo/nixos-hires.png"
+              width="500px"
+              alt="logo"
+            />
+          </a>
+        </p>
 
-                <p class="lead">
-                    <code>https://cache.nixos.org/</code> provides
-                    prebuilt binaries for Nixpkgs and NixOS. It is
-                    used automatically by the Nix package manager to
-                    speed up builds.
-                </p>
-            </div>
-            <hr />
-            <div class="help">
-                <p>
-                If you are having trouble, please reach out through one of the
-                <a href="https://nixos.org/nixos/support.html">support channels</a>
-                with the results of
-                <a href="https://github.com/NixOS/nixos-org-configurations/blob/master/terraform/cache/diagnostic.sh">this diagnostics script</a>
-                which will help us figure out where the issue lies.
-                </p>
-                <p>
-                For questions, or support, <a href="https://nixos.org/nixos/support.html">
-                the support page</a> from the NixOS website describes how to get in touch.
-                </p>
-            </div>
-        </div>
-    </body>
+        <p class="lead">
+          <code>https://cache.nixos.org/</code> provides prebuilt binaries for
+          Nixpkgs and NixOS. It is used automatically by the Nix package manager
+          to speed up builds.
+        </p>
+      </div>
+      <hr />
+      <div class="help">
+        <p>
+          If you are having trouble, please reach out through one of the
+          <a href="https://nixos.org/nixos/support.html">support channels</a>
+          with the results of
+          <a
+            href="https://github.com/NixOS/nixos-org-configurations/blob/master/terraform/cache/diagnostic.sh"
+          >this diagnostics script</a>
```

Comment:
The link is still here, no?

Reply:
Sorry I meant https://tarballs.nixos.org/

Reply:
Yeah, so it was not a regression in this changeset. Feel free to send a PR to update the links!